contribute/design/1.x/apis/cstorvolume_policy.md

```diff
@@ -53,29 +57,24 @@ type CStorVolumeConfig struct {
 	// Status represents the current information/status for the cstor volume
 	// claim, populated by the controller.
 	Status         CStorVolumeConfigStatus `json:"status"`
-  // VersionDetails contains version related info
-  VersionDetails VersionDetails         `json:"versionDetails"`
+	VersionDetails VersionDetails          `json:"versionDetails"`
 }
 
 // CStorVolumeConfigSpec is the spec for a CStorVolumeConfig resource
 type CStorVolumeConfigSpec struct {
 	// Capacity represents the actual resources of the underlying
 	// cstor volume.
 	Capacity corev1.ResourceList `json:"capacity"`
-	
-  // ReplicaCount represents the desired replica count for the underlying
+	// ReplicaCount represents the actual replica count for the underlying
 	// cstor volume
-	ReplicaCount int `json:"ReplicaCount"`
-
+	ReplicaCount int `json:"replicaCount"`
```

Comment:
should this be removed now, as it became part of CVP.Provision?

Reply:
As discussed this property moved to `CStorVolumeConfig.Provision.ReplicaCount`